README.md

```diff
@@ -288,10 +288,21 @@ fails to initialise.  In order to abort writing of that file instead, set in the
 
 The filewriter can use HDF's Single Writer Multiple Reader feature (SWMR).
 
+SWMR-mode writing is enabled by default.
+
+To disable SWMR, add to the `FileWriter_new` command:
+
+```json
+{
+  "use_hdf_swmr": false
+}
+```
+
 To write, as well as to read HDF files which use the SWMR feature requires at
-least HDF version 1.10.  This means that also tools like HDFView are required
+least HDF5 version 1.10.  This means that also tools like HDFView are required
 to link against at least HDF 1.10 in order to open HDF files which were written
-in SWMR mode.
+in SWMR mode.  One can also use the HDF5 tool `h5repack` to convert the file
```

Comment:
Does `h5repack` require some specific option? If so, can you add a simple example?

Reply:
The usage of `h5repack` is summarised in `h5repack --help`. Added comment to readme.